### PR TITLE
CHESS-1 Create project

### PR DESCRIPTION
**Pull request by zeljkosoko**

### CODE REVIEW

File: chess-1/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
+    "@testing-library/user-event": "^7.2.1",
+    "eslint": "^6.8.0",
+    "husky": "^4.2.0",
+    "lint-staged": "^10.0.2",
+    "prettier": "^1.19.1",
+    "prettier-standard": "^16.1.0",
```
Comment (ban17) [inline chess-1/package.json]: This need to be moved to dev dependencies

File: chess-1/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-scripts": "3.3.0",
+    "styled-components": "^5.0.0",
+    "stylelint": "^13.0.0",
+    "stylelint-config-rational-order": "^0.1.2",
+    "stylelint-order": "^4.0.0"
```
Comment (ban17) [inline chess-1/package.json]: To dev dependencies.

File: chess-1/.gitignore [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
```
Comment (ban17) [inline chess-1/.gitignore]: add .idea and .vscode
you can remove comments

File: chess-1/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```
Comment (ban17) [inline chess-1/README.md]: you can empty this file and instead all content just put project name and one line description.

File: chess-1/package.json (hunk 1 above)
Comment (zeljkosoko) [reply chess-1/package.json]: Moved.

File: chess-1/package.json (hunk 2 above)
Comment (zeljkosoko) [reply chess-1/package.json]: Moved.

File: chess-1/.gitignore (hunk 3 above)
Comment (zeljkosoko) [reply chess-1/.gitignore]: Added.

File: chess-1/README.md (hunk 4 above)
Comment (zeljkosoko) [reply chess-1/README.md]: readme changed.

**Pull request merged**